app/src/main/java/io/homeassistant/companion/android/widgets/media_player_controls/MediaPlayerControlsWidgetConfigureActivity.kt

```diff
@@ -182,10 +185,10 @@ class MediaPlayerControlsWidgetConfigureActivity : BaseActivity() {
         }
     }
 
-    private val entityDropDownOnItemClick =
+    /*private val entityDropDownOnItemClick =
```

Comment:
Instead of commenting out code, I think this can be completely removed.

Reply:
Sure